packages/content-management/content_editor/src/components/editor_flyout_content.tsx

```diff
@@ -177,7 +193,7 @@ export const ContentEditorFlyoutContent: FC<Props> = ({
                   onClick={onClickSave}
                   data-test-subj="saveButton"
                   fill
-                  disabled={isSubmitted && !form.isValid}
```

Comment:
Looks like we lost the `isSubmitted && !form.isValid` condition here. So the final condition for the disabled button should probably be something like:  `(isSubmitted && !form.isValid) || !hasNoChanges() `

Reply:
Done, thank you! ✅